src/System.IO.UnmanagedMemoryStream/tests/System.IO.UnmanagedMemoryStream.Tests.csproj

```diff
@@ -20,7 +21,6 @@
     <DefineConstants>TRACE;DEBUG</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
```

Comment:
To make this project test project consistent with others can you simply delete all the properties in these two groups and make those configurations empty and so they will pick-up the common properties from the dirs.props that is imported.